test/test_lib/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "test_lib",
+  "version": "1.0.0",
+  "description": "",
+  "main": "send-request-encrypted.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "Mobile Coin",
+  "license": "ISC"
```

Comment:
This should be GPL or MIT. Thoughts, @sugargoat ?

Reply:
Thank you! Yes, MIT would be preferred!